Small update for issue around golden images (#76775)
Azure/azure-content: articles/virtual-desktop/troubleshoot-vm-configuration.md

To learn more about this policy, see [Allow log on through Remote Desktop Services](/windows/security/threat-protection/security-policy-settings/allow-log-on-through-remote-desktop-services).

> ## Issues with golden image

Reviewer: ```suggestion
## I can't deploy the golden image
```

Author: Same as above - golden image and base image are different.

> Having the Azure Virtual Desktop agent as part of the golden image is not supported. The agent must always be installed once the golden image is being deployed and not as part of the golden image itself.

Reviewer: ```suggestion
Golden images don't currently include or support the Azure Virtual Desktop agent. You can only install the agent after you deploy the golden image.
```

Author: Hi Heidi, the golden image is different from the base images from Azure marketplace. A golden image is a custom image IT admins create that is used for their AVD deployments.
